__tests__/app.test.js

```diff
@@ -45,25 +37,141 @@ describe('/api', () => {
     });
 });
 
-describe('/api/articles/:article_id', () => {
+describe('GET: /api/articles/:article_id', () => {
     test('GET: 200 an article object with all associated properties', () => {
         return request(app)
         .get('/api/articles/1')
         .expect(200)
         .then((response) => {
-            const {rows} = response.body
-            rows.forEach(row => {
-                expect(row).toMatchObject({
+            const {article} = response.body
+            expect(article.article_id).toBe(1)
+            expect(article).toMatchObject({
+                created_at: expect.any(String)
+            })
+        })
+    })   
+});
+
+describe('GET: /api/articles', () => {
+    test('GET: 200 returns an array of all objects within articles without body property, sorted in descending order by creation date', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then((response) => {
+            const {articles} = response.body
+            expect(articles.length).toBe(37)
```

Comment:
You've had to change this test because you've changed the seed data. This shouldn't be the case.